docs/environment_variables.md

```diff
@@ -175,3 +194,5 @@ python -c "import httpx; httpx.get('http://example.com')"
 python -c "import httpx; httpx.get('http://127.0.0.1:5000/my-api')"
 python -c "import httpx; httpx.get('https://www.python-httpx.org')"
 ```
+
+
```

Comment:
For tidiness we could do without the trailing newlines here.

Reply:
Get it. This makes more sense. Thank you for this good suggestion.